public/Lab-Files/Lesson03/Challenges/wild-animal.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+<title> Wild Animal</title>
+
+
+</head>
+<body  
+
+    style="background-color:#d09632;">
+    
+     <h1 style="font-family: Pragmatica web;" style="color:#858b26;" align="center" >The Coolest Creature </h1>
+     
+     <hr>
+
+<p style="color:green;" align="center" >
+    <em>The African Lion</em> is one of the big cats in the genus Panthera and a member of the family Felidae.<br> 
+    The commonly used term <em>African lion </em>collectively denotes the several subspecies in Africa.<br/> <strong>Wikipedia
+Scientific name:</strong> <em>Panthera leo</em>
+<strong>Lifespan:</strong> 10 – 14 years (Adult, In the wild) <br/>
+<strong>Conservation status:</strong> Vulnerable (Population decreasing) Encyclopedia of Life <br/>
+<strong>Trophic level: Carnivorous Encyclopedia of Life<br/>
+Mass: Male:</strong> 420 lbs (Adult), <strong>Female:</strong> 280 lbs (Adult) <br/>
+<strong>Did you know: </strong>A small population of lions lives in the Gir Forest Sanctuary of India. 
+</p>
+<p>
+    <div style="background-color:black;color:white;padding:50px; font-size:180%">
+   <h2>About the African Lion</h2> 
+   <img src="wild-lion.jpg" alt="Lion catching a zibra" >
+
+Lions are the only cats that live in groups, which are called prides. 
+Prides are family units that may include up to three males, a dozen or so females, and their young.
+All of a pride's lionesses are related, and female cubs typically stay with the group as they age. 
+Young males eventually leave and establish their own prides by taking over a group headed by another male. <br/>
+    <img src="wild-lion02.jpg" alt="African lion with mouth open"  style="width:600px;height:480px" >
+    </p>
+    
+</body>
+</html>
```

Comment:
For this challenge, you were supposed to add the style tag to the head of the document, and avoid inline styling. However all of your css appears to be inline. 
```
<head>
<style type="text/css">
body {
    background-color: linen;
}
h1 {
    color: red;
    margin-left: 40px;
}
</style>
</head>
```
Something like that is using internal styling without it being inline. 
Also, please make sure you work on cleaning up your code and indenting. It will be easier to make changes when it looks organized.